recipe/meta.yaml

```
@@ -10,16 +10,16 @@ source:
   sha256: a14c1c5143081701d4fe62283e09cc5ad5336dff3a152de64eb77ee3180e9792
 
 build:
-  number: 0
+  number: 1
   noarch: python
 
 requirements:
   host:
     - python >=3.8
   run:
     - python >=3.8
-    - dask-core {{ version }}
-    - distributed {{ version }}
+    - dask-core ={{ version }},!={{ version }}a*
+    - distributed ={{ version }},!={{ version }}a*
```

This may need more tinkering, but this is one idea

```suggestion
    - dask-core =={{ version }}|{{ version }}.*,!={{ version }}a*
    - distributed =={{ version }}|{{ version }}.*,!={{ version }}a*
```